pkg/k8s/clusterresourcefactory.go

```diff
@@ -142,11 +141,21 @@ func (factory *ClusterResourceFactoryOptions) GetCloudProvider(verbose bool) (aw
 		klog.Error("Fail to get caller identity. Could you please validate the credentials?")
 		return nil, err
 	}
-	if verbose {
```

Comment:
Was there a reason to remove this for verbose output?

Reply:
It just wasn't helpful when debugging and felt like clutter that was leftover. That said there were effectively 0 additional logs emitted by verbose output on the entire `osdctl account console/cli` happy path, so I'm for adding more verbose logging to osdctl in general because it took quite a bit of digging for me to piece things together.

I think it should be more along the lines of "successful response from sts GetCallerIdentity" or "detected this is in GovCloud", etc.